src/WorkItemMigrator/tests/Migration.Jira-Export.Tests/RevisionUtils/FieldMapperUtilsTests.cs

```diff
@@ -409,6 +409,20 @@ public void When_calling_map_rendered_value_with_null_arguments_Then_and_excepti
             Assert.Throws<ArgumentNullException>(() => { FieldMapperUtils.MapRenderedValue(null, null, false, null, null); });
         }
 
+        [TestCase("2|hzyxfj:", 1088341183.0)]
+        [TestCase("2|hzyxfj:rx4", 1088341183.36184)]
+        public void When_calling_map_lexorank_value_with_valid_argument_Then_max_value_is_returned(string lexoRank, decimal expectedRank)
```

Comment:
Should this be "then the correct value is returned"?

Reply:
Yes it should. Will update that when in the office.

Reply:
This should be updated now